src/compiler/declarationEmitter.ts

```diff
@@ -709,7 +709,12 @@ namespace ts {
         function writeModuleDeclaration(node: ModuleDeclaration) {
             emitJsDocComments(node);
             emitModuleElementDeclarationFlags(node);
-            write("module ");
+            if (node.flags & NodeFlags.Namespace) {

```

Comment:
By the way, before I realized you fixed this issue, I found that this is the only place we ever use `NodeFlags.Namespace`. Just kind of weird.